src/store/state/state-model.ts

```diff
@@ -217,7 +218,7 @@ export const stateModel: StateModelIF = {
   continuationIn: {
     businessHomeValid: false,
     continuationAuthorization: null,
-    existingBusinessInfo: null
+    existingBusinessInfo: {} as ExistingBusinessInfoIF
```

Comment:
This allows us to re-reference object properties without an exception, though we might have to `$set` them in the object for reactivity.